frontend/components/LogDestinationIndicator/LogDestinationIndicator.tsx

```diff
@@ -48,10 +50,12 @@ const LogDestinationIndicator = ({
 
   const tooltipText = () => {
     switch (logDestination) {
+      case "null":
+        return `Each time a query runs, the data is received <br />
+            by the Fleet server and discarded.`;
       case "filesystem":
-        return `Each time a query runs, the data is sent to <br />
```

Comment:
This was an incorrect tooltip AFAICS.

Reply:
do we want this feature in the UI if it's only used for testing?

Reply:
It wouldn't hurt AFAICS, otherwise what do we show the load test user :)